src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -539,7 +535,7 @@ public void ReplyForbiddenWithWwwAuthenticateHeader(IEnumerable<string> scopes,
                 headers.Remove(HeaderNames.WWWAuthenticate);
             }
 
-            headers.Add(HeaderNames.WWWAuthenticate, v);
+            headers.Add(HeaderNames.WWWAuthenticate, new StringValues($"Bearer {parameterString}"));
         }
```

Comment:
this has been changed as well in latest master.

Reply:
Rebased.